client/src/js/references/components/Detail/__tests__/LatestBuild.test.js

```diff
@@ -0,0 +1,25 @@
+import { LatestBuild } from "../LatestBuild";
+
+describe("<LatestBuild />", () => {
+    let props;
+    props = {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['props' is never reassigned. Use 'const' instead.](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=4877812)